srcbpatch/jsonparser.h

```diff
@@ -0,0 +1,268 @@
+#pragma once
+#include <string_view>
+#include <utility>
+#include <variant>
+#include <vector>
+
+namespace bpatch
+{
+/// <summary>
+///   check if the symbol is in whitespaces
+/// </summary>
+/// <param name="sv">where to check</param>
+/// <param name="at">pos to check</param>
+/// <returns>true if the character is whitespace</returns>
+bool WhiteSpace(const std::string_view& sv, const size_t at);
```

Comment:
I suggest you to use a Verb in any function. It might be an 'action 'or just 'to be' verb